src/components/NewCommentForm.tsx

```diff
@@ -75,25 +160,39 @@
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', { 'is-danger': isErrorComment })}
+            value={comment}
+            onChange={handleCommentChange}
```

Comment:
It's a good practice to follow naming conventions for methods. So it can be `handleCommentInputChange` instead of `handleCommentChange`.